Automattic/vip
# feat(phpmyadmin): Add `vip  db phpmyadmin` command

## Description

In this PR, we are adding a new command `vip db phpmyadmin` which generates a URL to `phpmyadmin`, and opens in the default browser.

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run `./dist/bin/vip-db-phpmyadmin.js @6721.production`
1. Verify that it opened a URL with `/.wpvip/phpmyadmin/?token=...` prefix in your browser.
